indico/htdocs/js/custom/dropzone.js

```diff
@@ -94,6 +95,14 @@
                         if (options.submit_form) {
                             $form.submit();
                         }
+                        if (options.handleFlashes) {
+                            handleFlashes(response, true, $flashArea);
+                        }
+
+                        $dz.data('value', response.content);
+                        $form.find('.change-trigger').val($dz.data('value') ? 1 : 0);
+                        $form.trigger('indico:fieldsSaved', response);
+                        $form.trigger('change');
```

Comment:
a change event on a form? does that make much sense? if it's for the change tracker, then i'd say some logic is missing in the `indico:fieldsSaved` handler